terraform/environments/digital-prison-reporting/glue-connections.tf

```diff
@@ -0,0 +1,66 @@
+locals {
+  operational_db_jdbc_connection_string = "jdbc:postgresql://${module.aurora_operational_db.rds_cluster_endpoints["static"]}:${local.operational_db_port}/${local.operational_db_default_database}"
+  nomis_jdbc_connection_string          = "jdbc:oracle:thin:@${local.nomis_host}:${local.nomis_port}/${local.nomis_service_name}"
+}
+
+# Operational DataStore
+resource "aws_glue_connection" "glue_operational_datastore_connection" {
+  name            = "${local.project}-operational-datastore-connection"
+  connection_type = "JDBC"
+
+  connection_properties = {
+    JDBC_CONNECTION_URL    = local.operational_db_jdbc_connection_string
+    JDBC_DRIVER_CLASS_NAME = "org.postgresql.Driver"
+    SECRET_ID              = data.aws_secretsmanager_secret.operational_db_secret.name
+  }
+
+  physical_connection_requirements {
+    availability_zone      = data.aws_subnet.private_subnets_a.availability_zone
+    security_group_id_list = [aws_security_group.glue_job_connection_sg.id]
+    subnet_id              = data.aws_subnet.private_subnets_a.id
+  }
+}
+
+# Nomis
+resource "aws_glue_connection" "glue_nomis_connection" {
```

Comment:
Could you make these resources conditional, with a COUNT - able to manage per env Please

Reply:
@harichintala1 Yes sure. Should they be conditional on the `create_job` variable?